f2/src/locales/en.json

```diff
@@ -39,8 +39,6 @@
   "confirmCancelPage.title": "Cancel report?",
   "confirmationPage.ImpactTitle": "What could be affected?",
   "confirmationPage.ImpactTitle.edit": "Edit What could be affected",
-  "confirmationPage.anonymous.nag": "You are reporting anonymously. We did not ask for your name, email address or phone number.",
-  "confirmationPage.anonymous.title": "Anonymous report",
```

Comment:
just curious, were these supposed to be deleted?

Reply:
probably worked off a branch other than master. Merged some older PRs and its gone!